packages/dcos-diagnostics/windows.build.ps1

```diff
@@ -0,0 +1,14 @@
+$SRC_DIR = "c:\gopath\src\github.com\dcos\dcos-diagnostics\"
+new-item -itemtype directory "c:\gopath\src\github.com\dcos"
+
+copy-item -recurse  -Path c:/pkg/src/dcos-diagnostics/ -Destination c:/gopath/src/github.com/dcos/
+
+Push-Location $SRC_DIR
+
+$env:GOOS = "windows"
```

Comment:
Do we need to set `GOOS` when building on Windows?

Reply:
yes, when this was missing it was not building